tests/structure/io/test_pdb.py

```diff
@@ -77,6 +77,51 @@ def test_array_conversion(path, model, hybrid36, include_bonds):
     assert array1.coord.tolist() == array2.coord.tolist()
 
 
+@pytest.mark.parametrize(
+    "path, model",
+    itertools.product(glob.glob(join(data_dir("structure"), "*.pdb")), [None, 1, -1]),
```

Comment:
The number of parsed structure models does not influence the `CRYST1` record. Hence for the sake of test speed, I propose to remove the `model` parametrization from this test and always use `model=1` instead.